.circleci/config.yml

```diff
@@ -828,8 +828,9 @@ jobs:
             pulumi config set --path unchained:common.eks.nodeGroups[0].instanceTypes[3] r6i.4xlarge
             pulumi config set --path unchained:common.eks.nodeGroups[0].instanceTypes[4] r6id.4xlarge
             pulumi config set --path unchained:common.eks.traefik.autoscaling.enabled true
-            pulumi config set --path unchained:common.eks.traefik.autoscaling.cpuThreshold 20
+            pulumi config set --path unchained:common.eks.traefik.autoscaling.memoryThreshold 50
             pulumi config set --path unchained:common.eks.traefik.autoscaling.maxReplicas 20
+            pulumi config set --path unchained:common.eks.traefik.autoscaling.minReplicas 3
```

Comment:
is replica count ignored when using the auto scaler? feel like we should just use `5` to match the prev replica count

Reply:
it affects just the initial deployment